text/0000-component-boolean-arguments.md

```diff
@@ -0,0 +1,72 @@
+- Start Date: 11/30/2018
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Component boolean Arguments
+
+## Summary
+
+This RFC proposes to add a boolean argument to components, that aligns with boolean attribute of regular DOM elements.
+
+## Motivation
+
+Currently to pass a boolean property to a component a developer has to explicitly pass `true` as an argument.
+e.g.
+```hbs
+<Button @secondary={{true}} />
+```
+This does allow the value to be a variable e.g.
+
+```hbs
+<Dialog @visible={{this.visible}} />
+```
+
+If the value is not a variable `{{}}` can actually be ommited as a `true` string is truthy leading to somewhat easier to remember syntax of
+```hbs
+<MyHackyBoolArgument @bool=true />
+```
```

Comment:
this seems a little misleading to me as `@bool=false` will result in the `bool` property being the string `'false'`

Reply:
Added note about this. Thanks!